app/views/new_filters/_location_provider_filter.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="app-filter__group" data-qa="filters__area_and_provider">
+  <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+    <h3 class="govuk-fieldset__heading">
+      <% if @filters_view.provider_query? %>
+        Provider
+      <% else %>
+        Location
+      <% end %>
+    </h3>
+  </legend>
+  <p class="govuk-body-s govuk-!-margin-bottom-2" data-qa="area_or_provider_name">
+    <% if @filters_view.across_england_query? %>
+      Across England
+    <% elsif @filters_view.location_query? %>
+      <%= request.parameters['lq'] %>
+    <% elsif @filters_view.provider_query? %>
+      <%= request.parameters['query'] %>
+    <% end %>
```

Comment:
I think we need to pass these values through as hidden fields 

    <%= form.hidden_field(:query, value: params[:query]) %>
      <%= form.hidden_field(:lat, value: params[:lat]) %>
      <%= form.hidden_field(:lng, value: params[:lng]) %>
      <%= form.hidden_field(:loc, value: params[:loc]) %>
      <%= form.hidden_field(:lq, value: params[:lq]) %>

Reply:
Yep - that has been done now